i18n/en-US.json

```
@@ -614,20 +614,81 @@
 		},
 		"uploadPicture": "Upload a Picture",
 		"user": "User",
-		"call_forward": {
+		"callForwarding": {
+			"title": "Call Forwarding",
+			"strategies": {
+				"off": {
+					"title": "Off",
+					"helpText": "Call Forwarding is disabled and calls will not be forwarded."
+				},
+				"alwaysOn": {
+					"title": "Always On",
+					"helpText": "Calls will be forwarded to a destination of choice."
+				},
+				"busy": {
+					"title": "Busy",
+					"helpText": "Calls will be forwarded if all line appearances are busy."
+				},
+				"noAnswer": {
+					"title": "No Answer",
+					"helpText": "Calls will be forwarded if the call is not answered on any assigned devices."
+				},
+				"selective": {
+					"title": "Selective",
+					"helpText": "Calls will be forwarded based on predefined criteria, including specific phone numbers and schedules.",
+					"all": "All numbers",
+					"specific": "Specific numbers (10 max)" 
+				},
+				"common": {
+					"voicemail": {
+						"label": "Voicemail",
+						"selectLabel": "Target voicemail box"
+					},
+					"phoneNumber": {
+						"label": "Phone number",
+						"inputLabel": "Forward calls to"
+					},
+					"length": {
+						"always": "Always",
+						"custom": "Custom"
+					},
+					"options": {
+						"title": "Additional Options",
+						"forward": {
+							"title": "Forward direct calls only",
+							"helpText": "Inbound calls targeting ring groups, queues, etc will not be forwarded."
+						},
+						"acknowledge": {
+							"title": "Acknowledge Call",
+							"helpText": "Receive a prompt to accept forwarded calls. This prevents voicemails from being associated with the 'Forward To' number."
```

I wouldn't suggest using single quotes, instead you can do 
```suggestion
							"helpText": "Receive a prompt to accept forwarded calls. This prevents voicemails from being associated with the \"Forward To\" number."
```